### PR TITLE
🧑‍💻 DevOps - Provision Blob Storage Static Site

### LINKED ISSUE #1767: 🗃️ Archive Old Training Pages into /history
(Checked by @adamcogan @wicksipedia @bradystroud @jeoffreyfischer)

I have saved groups of pages as per https://github.com/SSWConsulting/SSW.Website/issues/1263 from the v1 site to HTML ready to be deployed to Blob Storage on `ssw.com.au/history`. 

```[tasklist]
### Tasks
- [x] Set up Front Door to route `ssw.com.au/history` to this Blob Storage
- [x] Create an `index.html` on each of the directories to access old pages (automatically via Python)
- [x] Link `/history` sitemap to main index sitemap
- [x] Breadcrumbs - Add `/history` to breadcrumbs links
- [x] Breadcrumbs - If clicked on link, navigate to old history page i.e. `/history/training`
- [x] Add box at top of page to denote that is an archived page (similar to archived rules)
- [x] Direct the "Book Now" buttons on pages e.g. `ssw/Training/BI-TheGuts.aspx` to `/booknow` (create temporary Next.js page that can be redirected from later)
- [x] Fix broken presenter image of Adam on `/Training` pages
- [x] Fix broken Adam Stephensen + Eric Phan photos
- [x] Make all resources (HTML + images) lowercase
- [x] Deploy training pages' HTML files to Blob Storage
- [ ] Test Please each group of pages with Wicksy, then Adam
```

### PR DESCRIPTION
**Pull request by Harry-Ross**
* Provisioning Azure Blob Storage for archive of v1 pages

- Affected routes: N/A

- Fixed #1767

### CODE REVIEW

File: infra/archiveStorage.bicep [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  'Standard_RAGRS'
+  'Standard_RAGZRS'
+  'Standard_ZRS'
+])
+param skuName string
+
+var unique = substring(uniqueString(resourceGroup().id), 0, 8)
+
+resource blobStorage 'Microsoft.Storage/storageAccounts@2022-09-01' = {
+  name: 'blobsswwebsite${unique}'
```
Comment (wicksipedia) [inline infra/archiveStorage.bicep]:
```suggestion
  name: 'stsswwebsite${unique}'
```
Change prefix for storage accounts - https://learn.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations

File: infra/archiveStorage.bicep [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+          allowedHeaders: [
+            '*'
+          ]
+          allowedMethods: [
+            'GET'
+            'HEAD'
+            'OPTIONS'
+          ]
+          allowedOrigins: [
+            '*'
```
Comment (wicksipedia) [inline infra/archiveStorage.bicep:68]: Do we want to scope origins down to our known url formats?

**Review by wicksipedia — APPROVED** [review]
Couple of minor changes, otherwise this looks like the QuickStart sample for blob storage + website

File: infra/archiveStorage.bicep (hunk 1 above)
Comment (Harry-Ross) [reply infra/archiveStorage.bicep]: Done 👍

File: infra/archiveStorage.bicep (hunk 2 above)
Comment (Harry-Ross) [reply infra/archiveStorage.bicep:68]: Moved to https://github.com/SSWConsulting/SSW.Website/issues/1834

**Pull request merged**